pkg/move/technologies.go

```diff
@@ -90,26 +94,28 @@ func copyByList(log logr.Logger, fs afero.Afero, from string, to string, paths [
 					log.Error(err, "failed to create new dir", "path", targetPath)
 
 					return err
-				} else {
-					log.V(1).Info("created new dir", "from", sourcePath, "to", targetPath, "mode", sourceStat.Mode())
 				}
-			} else {
-				log.V(1).Info("copying file", "from", sourcePath, "to", targetPath, "mode", sourceStat.Mode())
 
-				err = fsutils.CopyFile(fs, sourcePath, targetPath)
-				if err != nil {
-					log.Error(err, "error copying file")
+				log.V(1).Info("created new dir", "from", sourcePath, "to", targetPath, "mode", sourceStat.Mode())
 
-					return err
-				}
+				continue
 			}
 
+			log.V(1).Info("copying file", "from", sourcePath, "to", targetPath, "mode", sourceStat.Mode())
```

Comment:
hard to tell for sure, but again we change behavior here, which means we need to add unit-tests or revert this change.